<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplyMachineLearningWindow</class>
 <widget class="QMainWindow" name="ApplyMachineLearningWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2112</width>
    <height>1141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ML Task Scheduler[*]</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/app_resources.qrc">
    <normaloff>:/Icons/icons/Tango Icons/32x32/categories/preferences-desktop.png</normaloff>:/Icons/icons/Tango Icons/32x32/categories/preferences-desktop.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="settingsEditWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="opaqueResize">
            <bool>false</bool>
           </property>
           <property name="childrenCollapsible">
            <bool>false</bool>
           </property>
           <widget class="QGroupBox" name="groupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>300</height>
             </size>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QSplitter" name="splitter_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="opaqueResize">
                <bool>false</bool>
               </property>
               <property name="childrenCollapsible">
                <bool>false</bool>
               </property>
               <widget class="QWidget" name="verticalLayoutWidget">
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="font">
                    <font>
                     <bold>true</bold>
                     <underline>true</underline>
                    </font>
                   </property>
                   <property name="text">
                    <string>Main Options</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTreeView" name="mainOptionsTreeView">
                   <property name="rootIsDecorated">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="verticalLayoutWidget_2">
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="font">
                    <font>
                     <bold>true</bold>
                     <underline>true</underline>
                    </font>
                   </property>
                   <property name="text">
                    <string>General Options</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTreeView" name="generalOptionsTreeView">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="rootIsDecorated">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QSplitter" name="splitter_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="opaqueResize">
                <bool>false</bool>
               </property>
               <property name="childrenCollapsible">
                <bool>false</bool>
               </property>
               <widget class="QWidget" name="layoutWidget">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="font">
                    <font>
                     <bold>true</bold>
                     <underline>true</underline>
                    </font>
                   </property>
                   <property name="text">
                    <string>Model</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="indent">
                    <number>-1</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTreeView" name="modelOptionsTreeView">
                   <property name="rootIsDecorated">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="layoutWidget_2">
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="font">
                    <font>
                     <bold>true</bold>
                     <underline>true</underline>
                    </font>
                   </property>
                   <property name="text">
                    <string>Dataset</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTreeView" name="datasetOptionsTreeView">
                   <property name="rootIsDecorated">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="layoutWidget_3">
                <layout class="QVBoxLayout" name="verticalLayout_11">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="font">
                    <font>
                     <bold>true</bold>
                     <underline>true</underline>
                    </font>
                   </property>
                   <property name="text">
                    <string>Task</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTreeView" name="trainingOptionsTreeView">
                   <property name="rootIsDecorated">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="addToQueueButton">
          <property name="text">
           <string>Add to Queue</string>
          </property>
          <property name="icon">
           <iconset resource="../../res/app_resources.qrc">
            <normaloff>:/Icons/icons/Tango Icons/32x32/actions/format-indent-more.png</normaloff>:/Icons/icons/Tango Icons/32x32/actions/format-indent-more.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2112</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuasdf">
    <property name="title">
     <string>File...</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menusource">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Mode...</string>
    </property>
    <addaction name="actionSetLocalRunMode"/>
    <addaction name="actionSetNetworkRunMode"/>
   </widget>
   <widget class="QMenu" name="menuview">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuSet_Font_Size">
     <property name="title">
      <string>Font Size</string>
     </property>
     <property name="icon">
      <iconset resource="../../res/app_resources.qrc">
       <normaloff>:/Icons/icons/Tango Icons/32x32/apps/preferences-desktop-font.png</normaloff>:/Icons/icons/Tango Icons/32x32/apps/preferences-desktop-font.png</iconset>
     </property>
     <addaction name="actionIncreaseFontSize"/>
     <addaction name="actionDefaultFontSize"/>
     <addaction name="actionDecreaseFontSize"/>
    </widget>
    <addaction name="menuSet_Font_Size"/>
    <addaction name="actionReset_Splitters"/>
   </widget>
   <addaction name="menuasdf"/>
   <addaction name="menuview"/>
   <addaction name="menusource"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowIcon">
    <iconset resource="../../res/app_resources.qrc">
     <normaloff>:/Icons/icons/savesymbol.png</normaloff>:/Icons/icons/savesymbol.png</iconset>
   </property>
   <property name="windowTitle">
    <string>File Overview</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="100,100,0">
         <item>
          <widget class="QPushButton" name="saveCurrentConfigBtn">
           <property name="text">
            <string>Save</string>
           </property>
           <property name="icon">
            <iconset resource="../../res/app_resources.qrc">
             <normaloff>:/Icons/icons/Tango Icons/32x32/actions/document-save.png</normaloff>:/Icons/icons/Tango Icons/32x32/actions/document-save.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveCurrentConfigAsBtn">
           <property name="text">
            <string>Save As...</string>
           </property>
           <property name="icon">
            <iconset resource="../../res/app_resources.qrc">
             <normaloff>:/Icons/icons/Tango Icons/32x32/actions/document-save-as.png</normaloff>:/Icons/icons/Tango Icons/32x32/actions/document-save-as.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="SquareFrame" name="squareFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="OpenFileLocationBtn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../../res/app_resources.qrc">
                <normaloff>:/Icons/icons/Tango Icons/32x32/actions/document-open.png</normaloff>:/Icons/icons/Tango Icons/32x32/actions/document-open.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="FileExplorerView" name="ConfigFilePickerView"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowIcon">
    <iconset resource="../../res/app_resources.qrc">
     <normaloff>:/Icons/icons/Tango Icons/32x32/categories/applications-development.png</normaloff>:/Icons/icons/Tango Icons/32x32/categories/applications-development.png</iconset>
   </property>
   <property name="windowTitle">
    <string>Task Queue</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="OverlayWidget" name="MLQueueWidget">
       <property name="overlayHidden" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="UndoStack">
   <property name="windowIcon">
    <iconset resource="../../res/app_resources.qrc">
     <normaloff>:/Icons/icons/Tango Icons/32x32/actions/edit-undo.png</normaloff>:/Icons/icons/Tango Icons/32x32/actions/edit-undo.png</iconset>
   </property>
   <property name="windowTitle">
    <string>Undo Stack</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QUndoView" name="undoView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ConsoleDockWidget">
   <property name="windowTitle">
    <string>Command-Line Output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="OverlayWidget" name="ConsoleOverlayWidget">
       <property name="overlayHidden" stdset="0">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="ConsoleWidget" name="consoleWidget">
            <property name="ConsoleWidthPercentage" stdset="0">
             <number>88</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../../res/app_resources.qrc">
     <normaloff>:/Icons/icons/Tango Icons/32x32/actions/edit-undo.png</normaloff>:/Icons/icons/Tango Icons/32x32/actions/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo the last edit to the settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../../res/app_resources.qrc">
     <normaloff>:/Icons/icons/Tango Icons/32x32/actions/edit-redo.png</normaloff>:/Icons/icons/Tango Icons/32x32/actions/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionIncreaseFontSize">
   <property name="icon">
    <iconset resource="../../res/app_resources.qrc">
     <normaloff>:/Icons/icons/Tango Icons/32x32/actions/list-add.png</normaloff>:/Icons/icons/Tango Icons/32x32/actions/list-add.png</iconset>
   </property>
   <property name="text">
    <string>Increase Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionDefaultFontSize">
   <property name="icon">
    <iconset resource="../../res/app_resources.qrc">
     <normaloff>:/Icons/icons/Tango Icons/32x32/actions/view-refresh.png</normaloff>:/Icons/icons/Tango Icons/32x32/actions/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Default Size</string>
   </property>
  </action>
  <action name="actionDecreaseFontSize">
   <property name="icon">
    <iconset resource="../../res/app_resources.qrc">
     <normaloff>:/Icons/icons/Tango Icons/32x32/actions/list-remove.png</normaloff>:/Icons/icons/Tango Icons/32x32/actions/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Decrease Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../res/app_resources.qrc">
     <normaloff>:/Icons/icons/Tango Icons/32x32/actions/document-save.png</normaloff>:/Icons/icons/Tango Icons/32x32/actions/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="../../res/app_resources.qrc">
     <normaloff>:/Icons/icons/Tango Icons/32x32/actions/document-save-as.png</normaloff>:/Icons/icons/Tango Icons/32x32/actions/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionReset_Splitters">
   <property name="icon">
    <iconset resource="../../res/app_resources.qrc">
     <normaloff>:/Icons/icons/Tango Icons/32x32/actions/view-refresh.png</normaloff>:/Icons/icons/Tango Icons/32x32/actions/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Reset Splitters</string>
   </property>
  </action>
  <action name="actionSetLocalRunMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../res/app_resources.qrc">
     <normaloff>:/Icons/icons/Tango Icons/32x32/actions/go-home.png</normaloff>:/Icons/icons/Tango Icons/32x32/actions/go-home.png</iconset>
   </property>
   <property name="text">
    <string>Local</string>
   </property>
  </action>
  <action name="actionSetNetworkRunMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../res/app_resources.qrc">
     <normaloff>:/Icons/icons/Tango Icons/32x32/apps/internet-web-browser.png</normaloff>:/Icons/icons/Tango Icons/32x32/apps/internet-web-browser.png</iconset>
   </property>
   <property name="text">
    <string>Network</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConsoleWidget</class>
   <extends>QWidget</extends>
   <header>PySide6Widgets.Widgets.ConsoleWidget</header>
  </customwidget>
  <customwidget>
   <class>FileExplorerView</class>
   <extends>QTreeView</extends>
   <header>PySide6Widgets.Widgets.FileExplorerView</header>
  </customwidget>
  <customwidget>
   <class>OverlayWidget</class>
   <extends>QWidget</extends>
   <header>PySide6Widgets.Widgets.OverlayWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SquareFrame</class>
   <extends>QFrame</extends>
   <header>PySide6Widgets.Widgets.SquareFrame</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res/app_resources.qrc"/>
 </resources>
 <connections/>
</ui>
